tests/utils/fixtures/mock_directory.py

```diff
@@ -8,8 +8,9 @@ def __init__(self, id):
         self.object = "directory"
         self.id = id
         self.organization_id = "organization_id"
-        self.domain = "crashlanding.com"
-        self.name = "Ri Jeong Hyeok"
+        self.external_key = "ext_123"
```

Comment:
This external key is actually required. See our [API ref](https://workos.com/docs/reference/directory-sync/directory) (which isn't consistent, *sigh*) as well as our [API JSON model](https://github.com/workos/workos/blob/e084d3669e08a2eda7d3e8b4911668605fb9ead1/packages/api-json-models/src/rest-api/directory.ts#L10) and [Event JSON model](https://github.com/workos/workos/blob/8c93abedf98f81584088b56329dec0de9488e21f/packages/api-json-models/src/events/shared/directory.ts#L66).